templates/components/wysiwyg.build.hbs

```diff
@@ -4,7 +4,7 @@
   v-model.trim.lazy="value"
   ref="textarea"
   :name="name"
-  :id="name"
+  :id="tinymceId"
```

Comment:
This tells me that you're using the form component ID as the ID for the WYSIWYG field. This means that if you have 2 WYSIWYG fields in a single form, they would clash as they would have the same ID.

Reply:
In this case, a unique ID is generated and an ID is added to it by name, this ID is not conflicted, this was provided. This was the only solution that I have found, in order to 'Rich text' has to be initialised into different forms. If there are still options, I can revise and redo.

Reply:
This works 👍🏻